acisfp_check/acisfp_check.py

```diff
@@ -609,28 +607,32 @@ def draw_obsids(extract_and_filter,
     for eachobservation in obs_with_sensitivity:
         # extract the obsid
 
-        obsid = str(extract_and_filter.get_obsid(eachobservation))
+        obsid = extract_and_filter.get_obsid(eachobservation)
+        in_fp = eachobservation[extract_and_filter.in_focal_plane]
 
-        # Color all ACIS-S observations green; all ACIS-I 
-        # observations red
-        if eachobservation[extract_and_filter.in_focal_plane] == "ACIS-I":
-            color = 'red'
+        if obsid > 50000:
+            # ECS observations during the science orbit are colored blue
+            color = 'blue'
         else:
-            color = 'green'
+            # Color all ACIS-S observations green; all ACIS-I
+            # observations red
+            if in_fp == "ACIS-I":
+                color = 'red'
+            else:
+                color = 'green'
 
-        # Add the sensitivity text if the observation was found to be FP TEMP
-        # sensitive
-        #
-        # If the observation is FP sensitive in the first place............
-        if eachobservation[extract_and_filter.is_fp_sensitive]:
-            obsid = obsid + ' * FP SENS *'
+        obsid_txt = str(obsid)
+        # If this is an ECS measurement in the science orbit mark
+        # it as such
+        if obsid > 50000:
```

Comment:
Reading this through, I suppose this obsid method of determining the ECS observations was a bit of a surprise, but seems fine.  If we end up needing to use high number obsids for science you can obviously change this test.